### PR TITLE
bump versions for nightfall_code_scanner

### PR DESCRIPTION
**Pull request by visaals**
Following this example PR here: 

https://github.com/nightfallai/nightfall_dlp_action/pull/50/files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: nightfallDLP action step
-        uses: nightfallai/nightfall_dlp_action@v1.1.3
+        uses: nightfallai/nightfall_dlp_action@v1.1.4
```
Comment (evanfuller) [inline README.md:32]: can you bump our dogfood version too https://github.com/nightfallai/nightfall_dlp_action/blob/master/.github/workflows/nightfalldlp.yml

Comment (visaals) [reply README.md:32]: done!

Comment (visaals) [reply README.md:32]: wait do I need to do this in a separate PR? I can only reference 1.1.4 after I merge this PR and tag 1.1.4

Comment (visaals) [reply README.md:32]: looks like the build works after merging this PR 🥳 

https://github.com/nightfallai/nightfall_dlp_action/pull/53

**Pull request merged**